client/e2e/pages/authenticated/routes/page.spec.ts

```diff
@@ -50,119 +58,121 @@ function buildAccessLists(currentRole: UserRole): AppRoute[] {
 
 // 🏷 Annotate test suite as serial
 test.describe.configure({ mode: "serial" });
-// ➰ Loop through the entries of UserRole enum
-for (let [role, value] of Object.entries(UserRole)) {
-  role = "E2E_" + role;
-  const storageState = process.env[role + "_STORAGE"] as string;
-  test.describe(`Test Route Access for ${value}`, () => {
-    // 👤 Run test as this role
-    test.use({ storageState: storageState });
-    test("Test Navigate to Routes", async ({ page }) => {
-      // 🚨 Build routes allowed list for this user role
-      const accessLists = buildAccessLists(value);
+test.describe("Test Route Access", () => {
```

Comment:
If I understand this test correctly, it checks that every role can/can't get to the appropriate routes. If that's correct, then I'm not sure this test needs to check accessible routes because I think the role-based workflow tests will end up covering that. For example, the industry_user_admin has to go to dashboard > operations > operation in order to test the approve/deny boro id workflow

Reply:
@BCerki agree there is overlap but, I think there is value in having a separate test specific to access rights for all routes for all roles.